docs/user-guide/plugins/gke-provisioner.md

```diff
@@ -0,0 +1,64 @@
+# GKE Provisioner
+
+## Introduction
+This plugin streamlines the creation and configuration of a Google Kubernetes Engine (GKE) cluster on your Google Cloud Platform (GCP). It automates the provisioning process while implementing essential security measures, including a preconfigured firewall that allows access to SSH, HTTP (port 80), 8080, and Kubernetes NodePorts. By automating the GKE provisioning process through this plugin, you can save time, ensure consistency in cluster setup, maintain security standards, and provide a Kubernetes-ready environment for deploying your containerized applications. 
+
+{% hint style="warning" %}
+The GKE Provisioner plugin, while primarily recommended for Job Pipelines, but can also be effectively integrated into the PRE and POST stages of CI/CD pipelines.
+{% endhint %}
+
+### Prerequisites
+Before integrating the **GKE Provisioner** plugin make sure that you have an GCP account with valid permissions to provision GKE.
+
+---
+
+## Steps
+1. Go to Applications → **Devtron Apps**.
```

Comment:
We don't need an application to provision GKE, that's not the its purpose. Please go ahead with the job flow : )